driverapi/driverapi.go

```diff
@@ -103,6 +106,9 @@ type InterfaceInfo interface {
 
 	// AddressIPv6 returns the IPv6 address.
 	AddressIPv6() *net.IPNet
+
+	// SrcName return the srcName
+	SrcName() string
```

Comment:
That way this method could be part of `InterfaceNameInfo` where the set names method already is.